layers/state_tracker/cmd_buffer_state.h

```diff
@@ -180,12 +180,9 @@ class CMD_BUFFER_STATE : public REFCOUNTED_NODE {
     typedef uint64_t ImageLayoutUpdateCount;
     ImageLayoutUpdateCount image_layout_change_count;  // The sequence number for changes to image layout (for cached validation)
 
-    // Dynamic State
-    CBDynamicFlags status;          // Track status of various bindings on cmd buffer
-    CBDynamicFlags static_status;   // All state bits provided by current graphics pipeline
-                                    // rather than dynamic state
-    CBDynamicFlags dynamic_status;  // dynamic state set up in pipeline
-
+    // Track status of all vkCmdSet* calls, if 1, means it was set
+    CBDynamicFlags cmd_set_status_cb;        // for lifetime of CommandBuffer
+    CBDynamicFlags cmd_set_status_pipeline;  // for lifetime since last bound pipeline
```

Comment:
FFTI

What about something like this? Having them together in a struct IMO helps readability.

```suggestion
    struct Status {
        CBDynamicFlags cb;
        CBDynamicFlags pipeline;
    } set_status;
```

Reply:
:clap: agree, like this